atomdb/__init__.py

```diff
@@ -13,43 +13,75 @@
 # You should have received a copy of the GNU General Public License
 # along with AtomDB. If not, see <http://www.gnu.org/licenses/>.
 
-r"""AtomDB, a database of atomic and ionic properties."""
+"""AtomDB console script.
 
-import importlib
-from importlib.metadata import PackageNotFoundError
+This script provides a command-line interface to compile or query entries in the AtomDB database.
+"""
 
-from atomdb.periodic import Element
+import sys
+from argparse import ArgumentParser, ArgumentTypeError
+from atomdb import compile_species, load
 
-from atomdb.species import Species
 
-from atomdb.promolecule import Promolecule
+def positive_int(value):
+    """Validate if the argument is a positive integer."""
+    try:
+        ivalue = int(value)
+        if ivalue <= 0:
+            raise ArgumentTypeError(f"{value} is not a positive integer.")
+        return ivalue
+    except ValueError as e:
+        raise ArgumentTypeError(f"Invalid integer value: {value}") from e
 
-from atomdb.periodic import element_number, element_symbol, element_name
 
-from atomdb.species import compile_species, load, dump, raw_datafile
+def main():
+    """Main function for the AtomDB CLI."""
+    parser = ArgumentParser(
+        prog="atomdb",
+        description="Command-line tool to compile or query an AtomDB entry.",
+    )
 
-from atomdb.promolecule import make_promolecule
+    # Define mutually exclusive commands
+    command_group = parser.add_mutually_exclusive_group(required=True)
+    command_group.add_argument(
+        "-c", "--compile_species", action="store_true", help="Compile a species into the database."
+    )
+    command_group.add_argument(
+        "-q", "--query", action="store_true", help="Query a species from the database."
+    )
 
+    # Add arguments
+    parser.add_argument("dataset", type=str, help="Name of the dataset.")
+    parser.add_argument("elem", type=str, help="Element symbol (e.g., H, He, Li).")
+    parser.add_argument("charge", type=positive_int, help="Charge of the species (positive integer).")
+    parser.add_argument("mult", type=positive_int, help="Multiplicity of the species (positive integer).")
```

Comment:
Nice catch, about checking that the multiplicity be a positive number.

I think, however, that you can leave the `type` variable as `type=int` here and perform this verification at the level of the [`load`](https://github.com/theochem/AtomDB/blob/464f116f030da1ad05c28ae028e25985ecbed2ea/atomdb/species.py#L828) and `dump` functions in the species.py module. That way if someone tries to use the package as a script and inputs a negative multiplicity, we can throw an informative error message.